src/ViewModel/ContentHeaderProfile.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace eLife\Patterns\ViewModel;
+
+use Assert\Assertion;
+use eLife\Patterns\ArrayAccessFromProperties;
+use eLife\Patterns\ArrayFromProperties;
+use eLife\Patterns\SimplifyAssets;
+use eLife\Patterns\ViewModel;
+use Traversable;
+
+final class ContentHeaderProfile implements ViewModel
+{
+    use ArrayAccessFromProperties;
+    use ArrayFromProperties;
+    use SimplifyAssets;
+
+    private $details;
+    private $displayName;
+    private $miscLinks;
+    private $logoutLink;
+
+    public function __construct(string $displayName, array $logoutLink, array $miscLinks = [], array $details = [])
+    {
+        Assertion::notEmpty($displayName);
+        Assertion::notBlank($logoutLink);
+        Assertion::satisfy($details, function ($item) {
```

Comment:
`Assertion::satisfy` passes locally , but Jenkins is failing these assertions with `No assertion Assertion#satisfy exists`. https://ci--alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fpatterns-php/detail/PR-503/3/pipeline Any ideas @giorgiosironi @thewilkybarkid ?

Reply:
As this is a library we test it against the lowest and highest possible dependencies. The method was introduced in `beberlei/assert` 2.6.4 (yay, features in minor versions).

Reply:
Better to replace `$details` with `$affiliations` and `$emailAddress` anyhow.

Reply:
Bumping up the `composer.json` version dependency or replacing the assertion with custom code?